editor/src/uuiui/widgets/popup-list/popup-list.tsx

```diff
@@ -388,15 +390,24 @@ const MenuList = (props: MenuListComponentProps<SelectOption>) => {
   return <components.MenuList {...props} innerRef={ref} />
 }
 
-const DropdownIndicator = () => {
-  return <Icn category='controls/dropdown' type='carats' color='gray' width={11} height={11} />
+const DropdownIndicator: React.FunctionComponent<IndicatorProps<SelectOption>> = (
+  indicatorProps,
+) => {
+  return (
+    components.DropdownIndicator && (
+      <components.DropdownIndicator {...indicatorProps}>
+        <SmallerIcons.ExpansionArrowDown />
+      </components.DropdownIndicator>
+    )
+  )
 }
 
 const SingleValue = (props: SingleValueProps<SelectOption>) => {
+  console.log('single value props', props)
```

Comment:
Delete me please

Reply:
interesting. how could you push this? is someone not running the pre-commit hook?

Reply:
I think we bypassed it in an early commit because I needed to add some logging to show Omar. Does that mean that branch is poisoned, and _all_ future commits don't get run through the hook? How do you re-enable it?

Reply:
#ace17cf